build.gradle.kts

```diff
@@ -49,10 +67,10 @@ tasks.withType<Test> {
 
 gradlePlugin {
     plugins {
-        create("PlatformReleases") {
-            id = "edu.wpi.first.desktop.PlatformReleases"
+        register("PlatformReleases") {
```

Comment:
There are two plugins - one for jlink, and another for platform dependencies and releases. Both plugins should be exported, no?

Reply:
Supporting multiple plugins in 1 jar is actually a pain. Unless they've fixed that. But we really should be only exporting one plugin per jar. Then add the other plugin from setting up the primary plugin

Reply:
Then everything should be done from the platform releases plugin and the jlink plugin should be removed

Reply:
Yeah thats my thought. Basically the issue is in the plugins block there is no way to apply 2 plugins from the same id name. The ID should be `edu.wpi.first.desktop.PlatformReleases` and then the implementationClass be the main plugin class

Reply:
The Jlink plugin has applications outside of WPILIB.
Your welcome to host the Jlink plugin under this org:
https://github.com/GradleWeaver

I'll get everything setup.

I've been thinking of writing a Jlink plugin for a while now anyways.